src/Dock.Avalonia/DockControlState.cs

```diff
@@ -29,7 +29,7 @@ private void Enter(Point point, DragAction dragAction, IVisual relativeTo)
         {
             var isValid = Validate(point, DockOperation.Fill, dragAction, relativeTo);
 
-            if (isValid == true && _dropControl is DockPanel)
+            if (isValid == true && _dropControl is Panel)
```

Comment:
We probably need to add custom Panel type here or something otherwise dock target is displayed on tabitem header